drake/systems/controllers/InstantaneousQPController.h

```diff
@@ -0,0 +1,102 @@
+#ifndef INSTANTANEOUSQPCONTROLLER_H
+#define INSTANTANEOUSQPCONTROLLER_H
+
+#include <memory>
+#include "QPCommon.h"
+#include "drake/solvers/gurobiQP.h"
+#include "lcmtypes/drake/lcmt_qp_controller_input.hpp"
+
+#define USE_FASTQP 1
+#define GUROBI_OUTPUTFLAG 0
+#define GUROBI_METHOD 2
+#define GUROBI_PRESOLVE 0
+#define GUROBI_BARITERLIMIT 20
+#define GUROBI_BARHOMOGENEOUS 0
+#define GUROBI_BARCONVTOL (5e-4)
```

Comment:
Keep in mind that these macros are global; might want to give them more descriptive names (maybe prepend DRAKE_) to make sure we don't clobber somebody else's defines.